Earthfile

```diff
@@ -1,7 +1,30 @@
-VERSION 0.7
-FROM rust:latest
+VERSION 0.8
+FROM ubuntu:latest
```

Comment:
I'd love to think through keeping this base image as `rust` and using `python` for some of the python targets. Not that that would be the best way, I just feel like it has some advantages and would be simpler.

Earthly allows each target to inherit from whatever base image it needs, and it can pull files from any other image to do its work.

Reply:
That would be cool. We still need some unified tests for the whole workspace and compiling the python wrappers will require a python interpreter to be present